targets/ChibiOS/_nanoCLR/System.Device.Spi/sys_dev_spi_native_target.h

```
@@ -9,6 +9,7 @@
 #include <sys_dev_spi_native.h>
 #include <hal.h>
 #include <targetPAL.h>
+#include <targetPAL.h>
```

Clang (14.0.6)

[warning] duplicate include

(readability-duplicate-include)